src/common/Configuration/Config.cpp

```diff
@@ -209,7 +209,6 @@ bool ConfigMgr::LoadAppConfigs(std::string const& applicationName /*= "worldserv
     // #2 - Load .conf file
     if (!sConfigMgr->LoadMore(_initConfigFile))
     {
-        sLog->outString("");
```

Comment:
maybe this was supposed to print a new empty line?

Reply:
There are a lot of them, esp when loading worldserver, the use I cant understand

Reply:
```cpp
sLog->outString();
```

Reply:
but the reason of this empty sLog? it's just a newline or..?

Reply:
yes, empty line, cosmetic only